src/rt/array/construction.d

```diff
@@ -0,0 +1,165 @@
+/**
+ This module contains compiler support for constructing dynamic arrays
+
+  Copyright: Copyright Digital Mars 2000 - 2019.
+  License: Distributed under the
+       $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+     (See accompanying file LICENSE)
+  Source: $(DRUNTIMESRC rt/_array/_construction.d)
+*/
+module rt.array.construction;
+
+/**
+ * Does array initialization (not assignment) from another array of the same element type.
+ * Params:
+ *  to = what array to initialize
+ *  from = what data the array should be initialized with
+ * Returns:
+ *  The constructed `to`
+ * Bugs:
+ *  The functions `@safe` safety is faked.
```

Comment:
I think this sentence is too vague.  I suggest "This function template was ported from a much older runtime hook that bypassed safety, purity, and throwabilty checks.  To prevent breaking existing code, this function template is temporarily declared `@trusted` until the implementation can be brought up to modern D expectations."

Reply:
Thanks :)